react-emoji/src/components/EmojiContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import EmojiRow from './EmojiRow';
+import emoji from '../emoji.json';
+import './EmojiContainer.css'
+
+const emojiList = emoji;
```

Comment:
extra variable, could be simplified
```suggestion
import emojiList from '../emoji.json';
import './EmojiContainer.css'

```